modules/infra/cli/affinity.c

```
@@ -74,26 +73,25 @@ static cmd_status_t rxq_set(struct gr_api_client *c, const struct ec_pnode *p) {
+	gr_table_column(table, "CPU_ID", GR_DISP_RIGHT | GR_DISP_INT); // 0
+	gr_table_column(table, "IFACE", GR_DISP_LEFT); // 1
+	gr_table_column(table, "RXQ_ID", GR_DISP_RIGHT | GR_DISP_INT); // 2
+	gr_table_column(table, "ENABLED", GR_DISP_BOOL); // 3
 
 	gr_api_client_stream_foreach (q, ret, c, GR_AFFINITY_RXQ_LIST, 0, NULL) {
-		struct libscols_line *line = scols_table_new_line(table, NULL);
-		scols_line_sprintf(line, 0, "%u", q->cpu_id);
-		scols_line_sprintf(line, 1, "%s", iface_name_from_id(c, q->iface_id));
-		scols_line_sprintf(line, 2, "%u", q->rxq_id);
-		scols_line_sprintf(line, 3, "%u", q->enabled);
+		gr_table_cell(table, 0, "%u", q->cpu_id);
```

Cppcheck (2.20.0)

[warning] Uninitialized variable

(uninitvar)